graphql_api/types/mutation/save_terms_agreement/save_terms_agreement.graphql

```diff
@@ -8,4 +8,5 @@ input SaveTermsAgreementInput {
   businessEmail: String
   termsAgreement: Boolean!
   marketingConsent: Boolean
+  customerIntent: String!
```

Comment:
so when they call this mutation, they have to provide `customerIntent`, they can call this to update other things right?

Reply:
This form has 2 required things: termsAgreement + customerIntent, other fields are optional. We should validate we have all required fields to call this mutation successful + validate values of those if necessary, like with `customerIntent`

Reply:
@adrian-codecov I think this should be optional, it makes sense not to allow users form accessing dashboard if TOS is not approved, but even though we opt users in to choose customer intent, it should never be a blocker from accessing the dashboard / should not cause server error if it is missing, the mutation will handle this successfully in Gazebo as expected, let's allow client to control this field more and default things to `None`. This also means that we don't have to do risky deployments with gaps between client and API.